### ISSUE TITLE
Amazon login failing

### ISSUE THREAD

**Issue by xraken** [sammachin/alexaweb#2]
Hi, I got everything set up locally on an ubuntu lts and updated creds but when trying to log in from the login link (below the circle on the web page) I get to a page where Amazon asked me if it's Okay to authorize but when I click Ok I get this error page:

We're sorry!
An error occurred when we tried to process your request. Rest assured, we're already working on the problem and expect to resolve it shortly.
 Go to Amazon.com's Home Page

The URL of the error page looks like the token is being returned (deleted the token):
https://www.amazon.com/ap/oa?appActionToken=...extremely_long_string_deleted...&language=en_US&consentApproved=

I followed all of the instructions, what am I doing wrong? 
(BTW, the AlexaPi version worked fine and was able to retrieve and store the refresh token)

**Comment by xraken** [comment]
Figured it out.
I needed to change the "productID" value to my product id instead of the hard coded "alexaweb".
Next I ran into an issue with redis_url being undefined, initialized that to a random string (hope that's Ok).
Next issue:
class CodeAuthHandler(tornado.web.RequestHandler): ...callback = path+"code"...
should be callback = path+"/code" (i.e. "/" is missing) and this was making amazon server unhappy

I'm now running into an issue with posting the audio file, I will open a separate issue entry for that if needed.
